sdk/communication/Azure.Communication.CallAutomation/src/CallAutomationClient.cs

```diff
@@ -83,10 +83,10 @@ private CallAutomationClient(Uri endpoint, HttpPipeline httpPipeline, CallAutoma
             _pipeline = httpPipeline;
             _resourceEndpoint = endpoint.AbsoluteUri;
             _clientDiagnostics = new ClientDiagnostics(options);
-            AzureCommunicationServicesRestClient = new AzureCommunicationServicesRestClient(_clientDiagnostics, httpPipeline, endpoint.AbsoluteUri, options.ApiVersion);
```

Comment:
@minwoolee-msft this is the change which we discussed, resulted from "Endpoint"."format": "url" in swagger from [commit](https://skype.visualstudio.com/SCC/_git/sync_calling_concore-platform/commit/673e805f4502668a6baf4d765ae23b9dbdf70494?path=/Microsoft.Skype.Platform/Microsoft.Skype.Platform.ExecutionAgent/Azure.Communication.Service.ServerCalling/Swagger/ServerCallDocumentFilter.cs)